src/locales/fr.ts

```diff
@@ -243,6 +242,15 @@ const fr = {
       sampleData: {
         download: "Données d'échantillons",
       },
+      fileManifest: {
+        button: 'Manifeste',
+        title: 'Manifeste de fichier',
+        okText: 'Télécharger',
+        cancel: 'Annuler',
+        text: `Télécharger un manifeste des fichiers sélectionnés. Celui-ci pourra être utilisé par l'outil de téléchargement rapide des données du CQDG*. Ce manifeste comprend également des informations supplémentaires, notamment sur les participants et les échantillons associés à ces fichiers.`,
```

Comment:
Is `*` a typo ? Because it's not in the english message

Reply:
No, le * réfère à la ligne en dessous que j'ai oublié, merci !